From c7ef7eb3e6dda94eddd37f2fba04a19bf01036ea Mon Sep 17 00:00:00 2001
From: George Amanakis <gamanakis@gmail.com>
Date: Thu, 13 Jun 2019 00:03:38 -0400
Subject: [PATCH] mvebu: fixes commit 4089df4f4b0d

err_free_stats has been deprecated. Replace with err_netdev.

Compile-tested on: mvebu
Runtime-tested on: mvebu

Fixes: 4089df4f4b0d ("kernel: bump 4.14 to 4.14.125 (FS#2305 FS#2297)")
Signed-off-by: George Amanakis <gamanakis@gmail.com>
[altered hashes]
Signed-off-by: Koen Vandeputte <koen.vandeputte@ncentric.com>
(cherry picked from commit 1e3800df1808b4a5a977c3a9651958611df51c83)
---
 .../mvebu/patches-4.14/403-net-mvneta-convert-to-phylink.patch  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/linux/mvebu/patches-4.14/403-net-mvneta-convert-to-phylink.patch b/target/linux/mvebu/patches-4.14/403-net-mvneta-convert-to-phylink.patch
index 577317a84d..34a2d342db 100644
--- a/target/linux/mvebu/patches-4.14/403-net-mvneta-convert-to-phylink.patch
+++ b/target/linux/mvebu/patches-4.14/403-net-mvneta-convert-to-phylink.patch
@@ -905,7 +905,7 @@ Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
 +	phylink = phylink_create(dev, dn, phy_mode, &mvneta_phylink_ops);
 +	if (IS_ERR(phylink)) {
 +		err = PTR_ERR(phylink);
-+		goto err_free_stats;
++		goto err_netdev;
 +	}
 +
 +	pp->phylink = phylink;
-- 
2.22.0

